src/GovUk.Education.ExploreEducationStatistics.Admin/appsettings.Development.json

```diff
@@ -6,6 +6,7 @@
   "NotifyReleaseRoleTemplateId": "change-me",
   "NotifyPreReleaseTemplateId": "change-me",
   "NotifyContributorTemplateId": "change-me",
+  "NotifyReleaseApproversTemplateId": "change-me",
```

Comment:
You'll also need to make associated changes in `infrastructure/template.json` too - see *case-insensitive* examples of `NotifyReleaseRoleTemplateId` in the `infrastructure` folder.